proto/decentraland/renderer/kernel_services/friends_kernel.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package decentraland.renderer.kernel_services;
+
+enum FriendshipStatus {
```

Comment:
What if we move this enum to a friends_common.proto (or sth like that) file? Eventually, when we migrate the rest of the messages, we will likely need these statuses in a common file. What do you think?

Reply:
I cannot think of a case that renderer would need to send the friendship status to kernel, nor this enum being used in any other feature. Do you have any case that would be useful to move it?